content/en/docs/tasks/job/parallel-processing-expansion.md

```diff
@@ -67,19 +73,25 @@ Next, create all the jobs with one kubectl command:
 
 ```shell
 kubectl create -f ./jobs
-job "process-item-apple" created
-job "process-item-banana" created
-job "process-item-cherry" created
+```
+
+```
+job.batch/process-item-apple created
+job.batch/process-item-banana created
+job.batch/process-item-cherry created
 ```
 
 Now, check on the jobs:
 
 ```shell
 kubectl get jobs -l jobgroup=jobexample
-NAME                  DESIRED   SUCCESSFUL   AGE
-process-item-apple    1         1            31s
-process-item-banana   1         1            31s
-process-item-cherry   1         1            31s
+```
+
```

Comment:
please add The output is similar to this: as per the guidelines before any output